test/integration/clientHelper.ts

```diff
@@ -0,0 +1,76 @@
+import axios from "axios";
+import { HmacSHA1 } from "crypto-js";
+import { MatrixClient, MemoryStorageProvider, PantalaimonClient } from "matrix-bot-sdk";
+import config from "../../src/config";
+
+/**
+ * Register a user using the synapse admin api.
```

Comment:
Do I understand correctly that this file is designed primarily for testing?

If so, please mention this in the documentation. Ideally, we should also load this module only when we're running a tests, because otherwise it would extend the security perimeter. Is there any way to do so?

Also, I have the feeling that not all functions need to be exported.

Reply:
The file should not be loaded at all when Mjolnir is run normally, it is in the test directory and the only way for that to happen is if someone were to explicitly include it from something in `src`. There is also no way in the config to specify the registration shared secret either, so even if it was somehow included, it would not work. To be clear this is the registration endpoint that works with the shared secret and not an admin user.

Reply:
Oh, my bad, I hadn't seen that this was in `test`. Please ignore what I wrote :)